app/css/popup.css

```diff
@@ -24,15 +33,48 @@ body {
 }
 
 /**
- * Alert animations for ReactCSSTransitionGroup
+ * Alerts
  */
+.alert-success {
+  color: rgba(255, 255, 255, 0.9);
+  background: #56bd2d;
+  border: none;
+  border-radius: 4px;
+}
+
+@-webkit-keyframes alert-enter-animation {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
 
+@keyframes alert-enter-animation {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.alert {
+  -webkit-animation: alert-enter-animation 0.2s ease-in-out;
+          animation: alert-enter-animation 0.2s ease-in-out;
+}
+
+/**
+ * Alert animations for ReactCSSTransitionGroup
+ */
 .alert-exit {
   opacity: 1;
 }
 
 .alert-exit.alert-exit-active {
   opacity: 0.01;
+  -webkit-transition: opacity 400ms ease-in;
```

Comment:
Are these `-webkit` prefixes for Opera? Chrome and Firefox do not need them, they don't need to be added.

Reply:
These prefixes automatically added by Webpack, I'm compiling styles from SCSS.